docs/pipelines.md

```diff
@@ -1721,9 +1720,32 @@ Pipelines do not support the following items with custom tasks:
 
 For a better understanding of `Pipelines`, study [our code examples](https://github.com/tektoncd/pipeline/tree/main/examples).
 
+## Known Custom Tasks
+
+| Custom Task | Description |
+|:---|:--- |
+| [Task Loops][task-loops]| Runs a `Task` in a loop with varying `Parameter` values. 
+| [Pipeline Loops][pipeline-loops]| Runs a `Pipeline` in a loop with varying `Parameter` values.
+| [Common Expression Language][cel]| Provides Common Expression Language support in Tekton Pipelines.
+| [Wait][wait]| Waits a given amount of time, specified by a `Parameter` named "duration", before succeeding.
+| [Approvals][approvals]| Pauses the execution of `PipelineRuns` and waits for manual approvals.
+| [Pipelines in Pipelines][pipelines-in-pipelines]| Defines and executes a `Pipeline` in a `Pipeline`. 
+| [Task Group][task-group]| Groups `Tasks` together as a `Task`.
+| [Pipeline in a Pod][pipeline-in-pod]| Runs `Pipeline` in a `Pod`.
+
```

Comment:
Let's add a note at the end encouraging implementors of custom tasks to add their custom tasks to this table

Reply:
NP, done!